packages/now-next/package.json

```diff
@@ -28,11 +28,12 @@
     "@zeit/node-file-trace": "0.2.13",
     "async-sema": "3.0.1",
     "buffer-crc32": "0.2.13",
+    "execa": "2.0.4",
     "fs-extra": "7.0.0",
     "get-port": "5.0.0",
     "resolve-from": "5.0.0",
     "semver": "6.1.1",
-    "typescript": "3.5.2",
+    "typescript": "3.6.2",
```

Comment:
Why bump the version of TS? We should do this for all packages at once.

Reply:
Reverted it